pkg/processbaseline/processbaseline.go

```diff
@@ -1,9 +1,158 @@
 package processbaseline
 
-import "github.com/stackrox/rox/generated/storage"
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/protocompat"
+	"github.com/stackrox/rox/pkg/stringutils"
+)
+
+type keyPrefix string
+
+const (
+	deploymentContainerKeyPrefix keyPrefix = "DC"
+)
+
+var (
+	genDuration = env.BaselineGenerationDuration.DurationSetting()
+)
+
+func KeyToID(key *storage.ProcessBaselineKey) (string, error) {
```

Comment:
Moving this here was not necessary, but since I was moving other code to here, it seemed to make sense to move this here as well. Having this here means this does not need to import the datastore. I think it made sense to move things from the datastore to here that did not actually interact with the database.

Reply:
I will not do the refactoring for this ticket. I have created this ticket for the refactoring https://issues.redhat.com/browse/ROX-30341